core/zip/src/ZInflate.c

```diff
@@ -1129,10 +1129,10 @@ int R__unzip_header(int *srcsize, uch *src, int *tgtsize)
 
   /*   C H E C K   H E A D E R   */
   if (!(src[0] == 'Z' && src[1] == 'L' && src[2] == Z_DEFLATED) &&
-      !(src[0] == 'C' && src[1] == 'S' && src[2] == Z_DEFLATED) &&
-      !(src[0] == 'X' && src[1] == 'Z' && src[2] == 0)) {
-    fprintf(stderr, "Error R__unzip_header: error in header\n");
-    return 1;
+      !(src[0] == 'C' && src[1] == 'S' && src[2] == Z_DEFLATED) && !(src[0] == 'X' && src[1] == 'Z' && src[2] == 0) &&
+      !(src[0] == 'L' && src[1] == '4')) {
```

Comment:
This is becoming unwindly, can we create a named repository of those pairs to use here and the 2 or 3 other places.